src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,22 +15,76 @@ redirects:
+
+  <img
+    title="Apdex violations"
+    alt="A screenshot depicting critical violations in apdex"
+    src={apmCriticalViolationsinApdex}
+    />
+    </Step>
+    <Step>
+   ## Where does the problem lie? 
+
+    Scroll down to the **Infrastructure** section of the APM summary page. Here you'll see a table that lists each host connected to the `Billing Service` application and a record of their **Response time**, **Throughput**, **Error rate**, **CPU%**. and **Memory %**. Below the chart are histograms that highlight two of these golden signals. The default selections are **CPU %** and **Memory %** but you can also click the drop down menu in the top left and select a different view.
+
+    <img
+    title="toggle between histogram views"
+    alt="A screenshot depicting how you can toggle between histograom views"
+    src={apmToggleBetweenHistogramViews}
+    />
+
+    <figcaption>
+    You can toggle between different golden signals you want to inspect. 
+    </figcaption>
+
+    When you look at the CPU histogram, you can see that the CPU % for all of your hosts skyrocketed around 11:30 am. You can also see that this change in CPU occurred at the same time as a recent deployment. If you click on the [deployment marker](/docs/apm/apm-ui-pages/events/record-deployments/) it will tell you who released a change and what that change entailed. 
+
+    <img
+    title="CPU violation in APM"
+    alt="A screenshot depicting a CPU violation that corresponds to a deployment marker."
+    src={apmCPUPercentageViolationinAPM}
+    />
+  </Step>
+    <Step>
+     ## Dig deep into a specific host
+      
+      Now that you know that a recent deployment in your `Billing Service` application caused a spike in errors and critical Apdex violations you might want to look into a specific host for more clarity. Click the name of the host you want to inspect. It will reveal a sidebar that imports all relevant information from the **Infrastructure** page. This allows you to access all the information you need regarding your host and any service errors without leaving the rest of your data. 
+
+  <img
+    title="host with critical violations"
+    alt="A gif depicting a deeper inspection of a host with critical violations"
+    src={apmExploringaSpecificHostinAPM}
+  />
+
+  <figcaption>
+    Inspect your host without leaving the APM summary page. 
+  </figcaption>
+  </Step>
+</Steps>
+
+Now that you know how to troubleshoot with APM and infrastructure monitoring we're going to explore how to integrate APM and infrastructure data and put it into practice. 
 
 ## How to integrate APM and infrastructure data [#how-it-works]
```

Comment:
I would put this before the stepped tutorial IMO

Reply:
great idea